bagitobjecttransfer/recordtransfer/forms/user_forms.py

```diff
@@ -39,17 +53,19 @@ class UserProfileForm(forms.ModelForm):
         required=False,
     )
 
+    NAME_PATTERN = re.compile(r"^(?!.*[.\s']{2})[A-Za-z\s.'-]+$")
+    NAME_VALIDATION_MESSAGE = _(
+        "Name must contain only letters and may include these symbols: ' - . \n"
+    )
```

Comment:
This validation is too strict - a person may have accented characters in their name (e.g., é) and this would fail validation

Reply:
I've made the validation less strict by allowing these characters as well:
- `\u00C0-\u00D6`:  À to Ö.
- `\u00D8-\u00F6`:  Ø to ö
- `\u00F8-\u01FF`: ø to ÿ
For reference on Unicode characters: https://en.wikipedia.org/wiki/Latin-1_Supplement

Resolved by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/281/commits/70fde46c390930b972d9f5174fa95b75d94f5ee7